integration-tests/environments/electron/runner.js

```diff
@@ -0,0 +1,103 @@
+"use strict";
+
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+const { spawn } = require("child_process");
+const fs = require("fs");
+const path = require("path");
+const { MochaRemoteServer } = require("mocha-remote-server");
+
+const appPaths = {
+    darwin: "dist/mac/realm-electron-tests.app/Contents/MacOS/realm-electron-tests",
+    linux: "dist/linux-unpacked/realm-electron-tests",
+    win32: "dist/win-unpacked/realm-electron-tests.exe",
+};
+
+function determineSpawnParameters(processType, serverUrl) {
+    const platform = process.platform;
+    const appPath = path.resolve(appPaths[platform]);
+    if (fs.existsSync(appPath)) {
+        if (platform === "darwin") {
+            return {
+                command: appPath,
+                args: ["--", processType, serverUrl],
+            };
+        } else {
+            throw new Error(`Running tests on ${platform} is not supported yet`);
+        }
+    } else {
+        console.warn("🚧 Running an unpackaged version of the app 🚧");
```

Comment:
If this is a problem, why do we allow it instead of just throwing?

Reply:
It's not a problem in itself - you just have to know that this is not as close to a real-life scenario as we can get. The only reason we're not running packaged versions is that it's a little more involved and I would like to get that fixed when I setup running electron tests on Windows and MacOS too.